f5_openstack_agent/lbaasv2/drivers/bigip/agent_manager.py

```diff
@@ -563,8 +572,20 @@ def scrub_dead_agents_in_env_and_group(self, context):
     def update_operating_status(self, context):
         """Update pool member operational status from devices to controller."""
         if not self.plugin_rpc:
+            LOG.debug("update member status exits.")
+            return
+
+        if  self.member_update_interval == 0:
+            LOG.debug("update member status is disabled.")
+            return
+
+        now = datetime.datetime.now()
+        if (now - self.last_member_update).seconds < self.member_update_interval:
+            LOG.debug("update member status interval is not met.")
             return
```

Comment:
Maybe this task can not complete in 30s. Do we need another condition which prevents to start a new task, if there is an existing task running?